src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -175,6 +177,9 @@ public PlayerMock(ServerMock server, String name, UUID uuid)
 		setLocation(Bukkit.getWorlds().get(0).getSpawnLocation().clone());
 		setCompassTarget(getLocation());
 		closeInventory();
+
+		Random random = new Random();
```

Comment:
There is no reason to create a new `Random` instance here.
It would probably be best to just use the current Threads local random instance.